### ISSUE TITLE
Docs: document and/or prefer `css.configs.recommended`

### ISSUE THREAD

**Issue by fregante** [eslint/css#436]
### Docs page(s)

https://github.com/eslint/css#configurations

### What documentation issue do you want to solve?

It's not clear to me why the docs are using `extends` with a string array rather than the flat config's style of using `…css.configs.recommended`

Also it seems that the recommended config is incomplete, so that should be fixed as well: (it doesn't include the language and basic glob)
https://github.com/eslint/css/blob/e7a826ce30950f5f3d6b8261dbcc76924cf80057/src/index.js#L28-L33

### What do you think is the correct solution?

The expected configuration should be much simpler:

```js
import { defineConfig } from "eslint/config";
import css from "@eslint/css";

export default defineConfig([
	css.configs.recommended
]);
```

which matches the `eslint/js` and TS config format seen on https://typescript-eslint.io/getting-started:

```js
import js from '@eslint/js';
import { defineConfig } from 'eslint/config';
import tseslint from 'typescript-eslint';

export default defineConfig(
  js.configs.recommended,
  tseslint.configs.recommended,
);
```

### Participation

- [ ] I am willing to submit a pull request for this change.

### AI acknowledgment

- [x] I did not use AI to generate this issue report.
- [ ] (If the above is not checked) I have reviewed the AI-generated content before submitting.

**Comment by nzakas** [comment]
The example in the README is the officially recommended way to do configuration now. See https://eslint.org/blog/2025/03/flat-config-extends-define-config-global-ignores/

**Comment by fregante** [comment]
You might want to update the errors messages in eslint because this is what I get:

```
ESLint: 10.2.1

A config object is using the "extends" key, which is not supported in flat config system.

Instead of "extends", you can include config objects that you'd like to extend from directly in the flat config array.

If you're using "extends" in your config file, please see the following:
https://eslint.org/docs/latest/use/configure/migration-guide#predefined-and-shareable-configs

If you're not using "extends" directly (it may be coming from a plugin), please see the following:
https://eslint.org/docs/latest/use/configure/migration-guide#use-eslintrc-configs-in-flat-config
```

with this config:

```js
{
    files: ['**/*.css'],
    language: 'css/css',
    plugins: {css},
    extends: ['css/recommended'],
    rules: {
        'css/no-important': 'off',
    },
},
```

whereas this works:

```js
{
    ...css.configs.recommended,
    files: ['**/*.css'],
    language: 'css/css',
    rules: {
        ...css.configs.recommended.rules,
        'css/no-important': 'off',
    },
},
```

see commit and related failing CI: [#9321 (178c803d)](https://github.com/refined-github/refined-github/pull/9321/commits/178c803db17d360e1fdf668b9a0762af3f92f331)

**Comment by fregante** [comment]
This is due to the missing `defineConfig` call, but the error is still out of date / wrong. 

```diff
- export default [
+ export default defineConfig([
    cssConfigObject
- ];
+ ]);
```

**Comment by nzakas** [comment]
Please open an issue on eslint/eslint so we can address the error message.
